apps/web/modules/team/team-view.tsx

```diff
@@ -240,6 +240,4 @@ function TeamPage({ team, considerUnpublished, isValidOrgDomain }: PageProps) {
   );
 }
 
-TeamPage.isBookingPage = true;
```

Comment:
not needed because we now handle isBookingPage prop from server-side in app router

as done in #19330